### PR TITLE
Updated search template for programs

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-5316

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: course_discovery/apps/course_metadata/templates/search/indexes/course_metadata/program_text.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+{% for instructor in object.instructors %}
+    {{ instructor.full_name }}
+{% endfor %}
+
+{% for item in object.job_outlook_items.all %}
+    {{ item }}
+{% endfor %}
+
+{% for endorsement in object.individual_endorsements.all %}
+    {% include 'search/indexes/course_metadata/partials/endorsement.txt' %}
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/templates/search/indexes/course_metadata/program_text.txt:51]: Can you use this endorsement partial for corporate endorsements, too?

Comment (clintonb) [inline course_discovery/apps/course_metadata/templates/search/indexes/course_metadata/program_text.txt:51]: Different model. Look down.

Comment (rlucioni) [inline course_discovery/apps/course_metadata/templates/search/indexes/course_metadata/program_text.txt:51]: Hm, looks like it's only meant for individual endorsements.

**Comment by rlucioni** [comment]
👍

**Comment by schenedx** [comment]
👍

**Pull request merged**